composer.json

```diff
@@ -1,27 +1,26 @@
 {
-    "name": "mettle/sendportal-core",
-    "homepage": "https://github.com/mettle/sendportal-core",
+    "name": "sabas/sendportal-core",
+    "homepage": "https://github.com/sabas/sendportal-core",
```

Comment:
Please change these back

Reply:
Yeah I changed them to simplify the tests on my pc haha